ophyd/signal.py

```diff
@@ -1428,6 +1428,59 @@ def _pv_access_callback(self, read_access, write_access, pv):
             self._run_metadata_callbacks()
 
 
+class EpicsSignalNoValidation(EpicsSignalBase):
+    '''An EpicsSignal that does not verify values on set.
+    This signal does support readback, but does not guarantee that
+    the readback will match the set value.
+
+    Keyword arguments are passed on to the base class (Signal) initializer
+
+    Parameters
+    ----------
+    write_pv : str
+        The PV to write to
+    name : str, optional
+        Name of signal.  If not given defaults to write_pv
+    write_timeout : float or None, optional
+        The timeout for a reply when put completion is used. This is
+        only applied if the PV is connected within connection_timeout (below).
+
+        This is very different than the connection and read timeouts
+        above. It relates to how long an action takes to complete, such motor
+        motion or data acquisition. Any default value we choose here is likely
+        to cause problems---either by being too short and giving up too early
+        on a lengthy action or being too long and delaying the report of a
+        failure. A finite value can be injected here or, perhaps more usefully,
+        via `set` at the Device level, where a context-appropriate value can be
+        chosen.
+    connection_timeout : float or None, optional
+        Timeout for connection. This includes the time to search and establish
+        a channel.
+
+        The default value DEFAULT_CONNECTION_TIMEOUT means, "Fall back to
+        class-wide default." See EpicsSignalBase.set_defaults to
+        configure class defaults.
+
+        Explicitly passing None means, "Wait forever."
+    '''
+    def set(self, value, *args, **kwargs):
+        '''
+        Set the value of this signal, and return a completed Status
+        object, bypassing any readback verification
+
+        Returns
+        -------
+        st : Status
+            This status object will be finished
+        '''
+        self.put(value)
+
+        st = Status()
```

Comment:
One last small suggestion from me: link this back to the signal object for better str on the status
```suggestion
        st = Status(self)
```
ref https://github.com/bluesky/ophyd/blob/0b1570d08d6dd2616d041646171886ec737880db/ophyd/status.py#L571-L576